pkgs/development/python-modules/ryd-client/default.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+}:
+buildPythonPackage rec {
+  pname = "ryd-client";
+  version = "0.0.6";
+  pyproject = true;
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-PxrVdVw+dAkF8WWzYyg2/B5CFurNPA5XRNtH9uu/SiY=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [ requests ];
+
```

```suggestion
  # no tests
  doCheck = false;
  
  pythonImportsCheck = [ "ryd-client" ];
```
> - Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.
> - pythonImportsCheck is set. This is still a good smoke test even if pytestCheckHook is set.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines